packages/maskbook/src/extension/options-page/DashboardRouters/Container.tsx

```diff
@@ -150,77 +142,65 @@ const useStyles = makeStyles((theme) => {
             },
         },
         dividerCompact: {
-            [theme.breakpoints.down('sm')]: {
```

Comment:
already be checked: https://github.com/DimensionDev/Maskbook/pull/1912/files#diff-af89c552a9b1289c018f0ee44fa5c8f1726cb0acd8e5e4330fcd65db8a9d69f3R198